torch/csrc/jit/frontend/schema_type_parser.cpp

```diff
@@ -323,6 +324,11 @@ std::pair<TypePtr, c10::optional<AliasInfo>> SchemaTypeParser::parseType() {
     L.expect(')');
     alias_info = parseAliasAnnotation();
     value = DictType::create(key_type, value_type);
+  } else if( L.cur().text() == "torch.classes.cuda.Stream") {
```

Comment:
The annotation that you used has single quotes in it:
```
def stream(stream: 'torch.classes.cuda.Stream') -> None:
```
so I don't think this will evaluate to `true`.

Reply:
Oh you mean to say the quotes will also be considered as part of the str?